resources/translations/en.edn

```diff
@@ -170,15 +170,15 @@
          :apply "Apply"
          :apply-for-bundle "Apply for %1 resources as a bundle "
          :header "%1 resources in the cart"
-         :intro [:div.intro [:p "Add resources to the cart to apply for them. If several resources can be applied to with the same application, they will be automatically grouped into a bundle."]]
+         :intro [:div.intro [:p "Apply for access by adding resources to the cart. When applying for multiple rights at the same time, resources using the same application process are bundled together as one application."]]
```

Comment:
"application process" is not a REMS term. I like our old formulation more

Reply:
"multiple rights" does not sound correct semantically. "multiple resources" would be more coherent phrase.

Reply:
Resources aren't "applied for", access rights are. How about: 

"Apply for access by adding resources to the cart. When applying for access to multiple resources at the same time, resources using the same application are bundled together."

Reply:
Done